### ISSUE TITLE
Question: Town and nation claim block same nation

### ISSUE THREAD

**Issue by kiozix** [TownyAdvanced/Towny#4052]
hello is it normal that a city of a nation cannot claim next to another city of the same nation? blocked of 4 chunk how to remove this block

**Comment by LlmDl** [comment]
If you use the latest pre-release there is a config option that defaults to allow this behaviour.

**Comment by kiozix** [comment]
What name of option ?

**Comment by LlmDl** [comment]
Available in 0.96.1.10 and onwards:
```
  - New Config Option: town.min_distances_ignored_for_towns_in_same_nation
    - default: true
    - If true, the below settings: min_plot_distance_from_town_plot and min_distance_from_town_homeblock will be ignored for towns that are in the same nation. Setting to false will keep all towns separated the same.
    - Closes #3983.
```
